src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+`communication` because it is concerned with “intelligence” (Hegel 1991, 716).
+Intelligence, then, does not fall within the logical remit of `Mechanism`. If we
+just focus on the text, however, it looks as if intelligence has the effect of
+giving the form of universality to a content and thereby making it communicable.
+There are 2 logical steps to clarify here: (1) intelligence takes on a manifold
+content and gives it the form of universality – a capacity that intelligence
+has, and (2) intelligence is able to communicate the content _because_ it has
+the form of universality, _i.e._ a communicated content, whether it be through
+the `formal mechanical process` or through intelligence, must be universalised
+in order to be communicated.[^4]: It would be a fascinating thread of further
+research to explain exactly why intelligence has the capacity to give the form
+of universality to manifold content. I suspect that a good place to start for
+such an enquiry would be the ‘Philosophy of Spirit’.
+
+In contrast to intelligence, processes within the objective sphere, _i.e._ the
+`formal mechanical process` do not give to content the form of universality but
+are in-and-for-themselves universal. Why is this the case? The answer must have
+to do with the very beginning of `Mechanism` where the `mechanical object`.
+Hegel writes that “It is therefore in and for itself a universal -universality
+not in the sense of a community of properties, but a universality that pervades
+the particularity and in it is immediate individuality” (Hegel 1991, 711). The
+reason why the objective, and by extension the `formal mechanical process` is
+in-and-for-itself the universal is because the difference between the
+determinations of `Concept` have been effaced.[^5]: See:
+[first-paragraph](hegel/reference/mechanical-object/first-paragraph). As such,
+by virtue of being in-and-for itself the universal, and not needing to give the
+form of universality to content, the `communication` of the
+`formal mechanical process` has the virtue of being unstoppable. As we said
+above, it quite simply pervades the entire spiritual and/or material sphere. The
+individuality of objects that encounter it does not come into consideration: the
+law, morality, etc, do no alter according to the individual but are universal.
+Whilst the content of an intelligence can be given the form of a universal, the
+fact that it is not in-and-for-itself a universal means that its content can be
+resisted, _i.e._ I am not automatically within the remit of Kant’s conception of
+the moral law just because it has the form of universality – my individuality
+has the power to resist. Not so with the `formal mechanical process` -- whilst I
+can resist against the law by breaking the law or protesting against the law, I
+am nevertheless automatically implicated within the law. My specific actions or
+my particular character does not have the power to pull me outside of its
+gravitational pull.
```

Comment:
I think this is the wrong place to use analogies of gravity since it'll become thematized later in Mechanism. Can we think of other analogies more appropriate?

Reply:
I do think gravity works with communication for reasons given above, but I have deleted that sentence anyway because I made a mistake.